### DISCUSSION TITLE
JavaAgent HTTP params/headers Collector Pulgin，do not collect on the ExitSpan

### DISCUSSION THREAD

**Discussion by WinterWaterWarm** [apache/skywalking#9772] [Q&A] [answered]
Want to know why parameters and request headers are collected only on EntrySpan, considerations for this decision

If I want to do collection on HTTP ExitSpan, is there any better way to solve it?

**Comment by wu-sheng** [comment, marked as answer]
I think you are asking questions in the wrong way. There isn't `why` actually.
The previous contributors think it is better to collect on the server side. That is it. The open source project is made by whom contributed codes.

**Reply by WinterWaterWarm** [reply]
Okay, I get it！
Thanks for the answer.
